src/resources/i18n-target.html

```diff
@@ -0,0 +1,52 @@
+<template>
+
+  <button id="locale-changer-de" click.delegate="changeLocale('de')">DE</button>
+
+  <span id="i18n-simple" t="simple.text;[title]simple.attr"></span>
+
+  <span id="i18n-ctx" t="status"></span>
+  <span id="i18n-ctx-dispatched" t="status" t-params.bind="{context: 'dispatched', date: dispatchedOn}"></span>
+  <span id="i18n-ctx-delivered" t="status" t-params.bind="{context: 'delivered', date: deliveredOn}"></span>
+
+  <span id="i18n-items-0" t="itemWithCount" t-params.bind="{count: 0}"></span>

```

Comment:
I liked the idea that above the `id` is pretty much telling which feature we're testing. Maybe here we could include `count` or `plural` into the id.

Reply:
Done